.circleci/config.yml

```diff
@@ -37,10 +37,10 @@ jobs:
       - run:
           name: Insert key and flag data into demo
           command: |
-            sed -i "s/YOUR_SDK_KEY/${LD_HELLO_WORLD_SDK_KEY}/" hello.r
-            sed -i "s/YOUR_FEATURE_KEY/${LD_HELLO_WORLD_FLAG_KEY_WITH_TRUE_VALUE}/" hello.r
+            sed -i "s/SDK_KEY=*\".*\"/SDK_KEY=\"${LD_HELLO_WORLD_SDK_KEY}\"/" hello.r
```

Comment:
Please review this file especially carefully -- the tests are passing for this branch, but I'm very new to CircleCI configurations and would appreciate any feedback on this.